api/api-pipeline-operations-v1/src/main/java/com/thoughtworks/go/apiv1/pipelineoperations/representers/StageInstanceModelRepresenter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.apiv1.pipelineoperations.representers;
+
+import com.thoughtworks.go.api.base.OutputWriter;
+import com.thoughtworks.go.presentation.pipelinehistory.StageInstanceModel;
+
+public class StageInstanceModelRepresenter {
+    public static void toJSON(OutputWriter outputWriter, StageInstanceModel stageInstanceModel) {
+        if (stageInstanceModel.getResult() != null) {
+            outputWriter.add("result", stageInstanceModel.getResult().toString());
+        }
+        if (stageInstanceModel.getRerunOfCounter() == null) {
+            outputWriter.add("rerun_of_counter", "null");
```

Comment:
adding a string "null" will render it as a string in JSON (not at null value).

Reply:
#### Actual output:
`"rerun_of_counter" : "null"`

#### Expected output:
`"rerun_of_counter" : null`